index.ts

```diff
@@ -214,7 +214,23 @@ namespace tss {
         private formatDiagnostics(diagnostics: ts.Diagnostic[]): string {
             return diagnostics.map((d) => {
                 if (d.file) {
-                    return 'L' + d.file.getLineAndCharacterOfPosition(d.start).line + ': ' + d.messageText;
+                    const line       = d.file.getLineAndCharacterOfPosition(d.start).line;
+                    const linePrefix = `Line ${line}: `;
+
+                    if (typeof d.messageText === 'object') {
+                        if (d.messageText.next != null) {
+                            let indentStr = '';
+                            for (let i = 0; i < linePrefix.length; i++) {
+                                indentStr += ' ';
+                            }
+
+                            d.messageText.messageText += `\n${indentStr}${d.messageText.next.messageText}`;
+                        }
+
+                        d.messageText = d.messageText.messageText;
+                    }
+
+                    return linePrefix + d.messageText;
```

Comment:
Ah, I just opened a PR which does the same thing. #72 

Maybe that one is better because a method similar to this is [already provided in typescript](https://github.com/Microsoft/TypeScript/blob/24814ec387a5ccc3619be80f0fc4a79b776a4eee/src/compiler/program.ts#L244-L268)